[Outlook] (event-based activation) Add limitation on JS file support in Outlook on Windows (#4149)
OfficeDev/office-js-docs-pr: docs/outlook/autolaunch.md

1. Save your changes.

> [!NOTE]
> If you're developing an event-based add-in in Outlook on Windows, be aware of the following:
>
> - At present, imports aren't supported in the JavaScript file where you implement the handling for event-based activation.
> - Code included in the `Office.onReady()` and `Office.initialize` functions isn't processed. We recommend adding your add-in startup logic, such as checking the user's Outlook version, to your event handlers instead.

> > - Only a single JavaScript file is supported in an event-based add-in project. You must bundle your JavaScript code into a single file.

Reviewer: ```suggestion
> - In an event-based add-in project, only the single JavaScript file referenced in the ... node/element of the manifest is supported for event-based activation. You must bundle your event-handling JavaScript code into a single file.
```

- Ensure that the following conditions are met in your add-in's manifest.
  - Verify that your add-in's source file location URL is publicly available and isn't blocked by a firewall. This URL is specified in your manifest's [SourceLocation element](/javascript/api/manifest/sourcelocation).
  - Verify that the **\<Runtimes\>** element correctly references the HTML or JavaScript file containing the event handlers. Outlook on Windows uses the JavaScript file during runtime, while Outlook on the web and on new Mac UI use the HTML file. For an example of how this is configured in the manifest, see [Configure the manifest](#configure-the-manifest).

> - An event-based add-in that runs in Outlook on Windows only supports a single JavaScript in the add-in project. You must bundle your JavaScript code into a single file.

Reviewer: I'm unclear what "a single JavaScript" means. Can you clarify?

```suggestion
- An event-based add-in that runs in Outlook on Windows only supports a single JavaScript reference in the add-in project. You must bundle your JavaScript code into a single file.
```

Author: Thanks for catching that. That should say "single JavaScript file".

> - An event-based add-in that runs in Outlook on Windows only supports a single JavaScript file in the add-in project. You must bundle your JavaScript code into a single file.

Reviewer: ```suggestion
- An event-based add-in that runs in Outlook on Windows only supports a single JavaScript file referenced in the ...node of the manifest. You must bundle your event-handling JavaScript code into a single file.
```
